vespalib/src/vespa/vespalib/util/rcuvector.hpp

```diff
@@ -20,7 +20,8 @@ RcuVectorHeld<T>::~RcuVectorHeld() = default;
 template <typename T>
 size_t RcuVectorBase<T>::calcNewSize(size_t baseSize) const {
     size_t delta = (baseSize * _growStrategy.getGrowFactor()) + _growStrategy.getGrowDelta();
-    return baseSize + std::max(delta, static_cast<size_t>(1));
+    size_t newSize = baseSize + std::max(delta, static_cast<size_t>(1));
+    return std::max(newSize, _growStrategy.getMinimumCapacity());
```

Comment:
The method is misnamed, `calcNewCapacity` or `calc_new_capacity` is probably a better name than `calcNewSize`